src/porepy/numerics/linalg/matrix_operations.py

```diff
@@ -81,7 +102,6 @@ def merge_matrices(A, B, lines):
     ------
```

Comment:
Wouldn't it be safer to have two functions, replace_rows and replace_columns? I am extremely wary of assumptions on matrix format.

Reply:
Possibly, this is an old function that I just copied. I'll see what I can do with a reasonable effort.